charts/shoot-core/components/charts/kube-proxy/templates/conntrack-fix-script.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: conntrack-fix-script
+  namespace: kube-system
+  labels:
+    app: kubernetes
+    gardener.cloud/role: system-component
+    origin: gardener
+    role: proxy
+data:
+  conntrack_fix.sh: |
+    #!/bin/sh -e
+    trap "kill -s INT 1" TERM
+    apk add conntrack-tools
+    sleep 120 & wait
```

Comment:
I guess you want to wait for kube-proxy process to start-up and do something meaningful here, right? I guess it's "ok" for such a mitigation for now, but generally it can take longer than 2m for the kube-proxy to startup while this container is already running (e.g., delayed image pull). It would be better to have a check if the kube-proxy process is running already. If you have an idea how to easily achieve that? As said, I don't insist on it because it's just a mitigation, but I wanted to point it out.

Reply:
I could replace it with 
``` bash
until pgrep kube-proxy; do
  sleep 2 & wait
done
```
and set `shareProcessNamespace: true` in the pod

I think other ways for ipc are much more complicated.

WDYT?

Reply:
Yeah, I don't insist on this now, just wanted to point it out. If we find an easy solution then sure, otherwise I think we can still go ahead with such mitigation for now.

Your proposal looks good, but then you'd have to adapt the trap in line 15 as well, right (your script is not necessarily PID 1)? cc @istvanballok

Reply:
Sure, will do that. Should be sufficient to replace pid 1 with `$(pgrep "/bin/sh")`

Reply:
I discussed with @wyb1 and @istvanballok that we will leave it like this for now. We will concentrate on fixing this in `kube-proxy`.

Reply:
Sure, thanks. Merging then :)